usdf/datasets.rst

```diff
@@ -77,67 +93,19 @@ Here is a template for what each refcat's readme should contain:
     The configuration that was used to ingest the data is included in this
     directory as `IngestIndexedReferenceTask.py`.
```

Comment:
We can actually do something like the immutability/sharing below, as long as the modification access is coming from the interactive or batch nodes (not Kubernetes/notebooks).  But we can put that back in when it's better-defined.

Reply:
Isn't that basically what's described with setting permissions above?

Reply:
Changing ownership is kind of a brute-force means of doing this; we can do something a bit more sophisticated for "temporary unlock to allow modification" or even "permanently allow curation by this group of people other than rubinmgr".